store.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>Storage</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="stylesheet" href="css/store.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+    <link rel="icon" href="img/karma-logo.svg" />
+  </head>
+  <body>
+    <header>
+      <img src="img/karma-logo.svg" alt="logo" />
+      <nav>
+        <ul>
+          <li><a href="index.html">Meet Karma</a></li>
+          <li><a href="#">How it Works</a></li>
+          <li><a href="store.html">Store</a></li>
+          <li><a href="#">Blog</a></li>
+          <li><a href="#">Help</a></li>
+          <li><a href="#">Login</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="left-section">
```

Comment:
How about just having this section with the form in `<main>` and the other one in an `<aside>`?

Reply:
you mean I have to remove sections and leave form, add aside within main?

Reply:
The form is representing your dominant content of the page, so it makes perfectly sense to have it inside of `<main>`, but the right column is just secondary content, not really necessary, so semantically, it could live in an `aside`.

So you could have a structure like this:

```html
<main>
your form etc
</main>
<aside>
your ride column content
</aside>
```